dsp-schema/core/src/main/scala/dsp/schema/domain/SchemaCommands.scala

```diff
@@ -0,0 +1,49 @@
+package dsp.schema.domain
+
+import dsp.errors.ValidationException
+import dsp.valueobjects.LangString
+import dsp.valueobjects.Schema
+import zio.prelude.Validation
+
+import java.time.Instant
+
+case class SmartIri(value: String)
+
+sealed abstract case class CreatePropertyCommand private (
+  ontologyIri: SmartIri, // TODO: make SmartIri a ValueObject at some point
```

Comment:
Did you check if it is OK to not have a "real" `SmartIri` here? Not that there are some checks/validations that now can't be done anymore.

Reply:
it's true that this can't do all the validation we eventually need. But that is a responsibility of the `SmartIri` valueobject, which in its current form is only a placeholder, but that is out of scope of this task. However, before we can use the `CreatePropertyCommand` in a productive environment for more than validation, we need to address SmartIri first.